samples/net/sockets/http_server/src/server.c

```diff
@@ -0,0 +1,979 @@
+/*
```

Comment:
Please move this file to `subsys/net/lib/http/http_server.c` so that it can be built into Zephyr's HTTP server library and subsequently used by both samples and tests.